docs/man/git-lfs-faq.adoc

```diff
@@ -0,0 +1,144 @@
+= git-lfs-fae(5)
```

Comment:
Typo: `faq` not `fae`.

Reply:
Ah, yes, this is an open source project, not an RPG campaign.